Representable/representable
# Adding new site graphics

**changes**
- state page now has consistent graphic, replacing state svg
- entry page now has a graphic that helps explain what a map may look like

**to test**
- go to state page
- go to entry page
- check alignment, mobile, etc

**screenshots**
state page
![image](https://user-images.githubusercontent.com/41943646/96269322-3c892a80-0f98-11eb-9ce4-d6fe48126955.png)
entry page
![image](https://user-images.githubusercontent.com/41943646/96269418-57f43580-0f98-11eb-9e4a-eab8e68540ed.png)

note: keeping the state svg files in the repo for now, in case they prove useful later on
